docs/src/components/home/MuiStatistics.tsx

```diff
@@ -5,9 +5,9 @@ import Typography from '@mui/material/Typography';
 
 const data = [
   { title: '2.8M', metadata: 'Weekly downloads on npm' },
```

Comment:
https://npm-stat.com/charts.html?package=@material-ui/core,@mui/material

```suggestion
  { title: '3.2M', metadata: 'Weekly downloads on npm' },
```